src/app.dump.ts

```diff
@@ -45,35 +46,35 @@ export default async function dump({ env }: Parameters) {
     if (Deno.env.get("TEA_REWIND")?.includes("TEA_PREFIX")) {
       // we do this manually because we also set it for stuff executed through tea
       // which means when developing tea this otherwise doesn’t get unset lol
-      await print(unsetEnv("TEA_PREFIX"))
+      await usePrint(unsetEnv("TEA_PREFIX"))
```

Comment:
From an API perspective, I feel like `usePrint()` should return an object containing a `print()` function. I believe it's generally the case that hooks are of a form of `function useSomething(optional_initial_state): object_of_functions_and_values`. At least in `cli`'s codebase.

Reply:
Ironically, such minor complexity would reduce your diff size, though not very meaningfully.